<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MobileSettingsDialog</class>
 <widget class="QWidget" name="MobileSettingsDialog">
  <property name="geometry">
   <rect>
	<x>0</x>
	<y>0</y>
	<width>467</width>
	<height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qutIM Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
	<number>0</number>
   </property>
   <property name="spacing">
	<number>0</number>
   </property>
   <item row="0" column="0">
	<widget class="QTabWidget" name="tabWidget">
	 <property name="tabShape">
	  <enum>QTabWidget::Triangular</enum>
	 </property>
	 <property name="currentIndex">
	  <number>0</number>
	 </property>
	 <property name="documentMode">
	  <bool>false</bool>
	 </property>
	 <widget class="QWidget" name="tabWidgetPage1">
	  <attribute name="title">
	   <string>General</string>
	  </attribute>
	  <layout class="QVBoxLayout" name="verticalLayout">
	   <item>
		<widget class="QListWidget" name="generalListWidget">
		 <property name="frameShape">
		  <enum>QFrame::NoFrame</enum>
		 </property>
		</widget>
	   </item>
	  </layout>
	 </widget>
	 <widget class="QWidget" name="tabWidgetPage2">
	  <attribute name="title">
	   <string>Protocols</string>
	  </attribute>
	  <layout class="QGridLayout" name="gridLayout_2">
	   <item row="0" column="0">
		<widget class="QListWidget" name="protocolListWidget">
		 <property name="frameShape">
		  <enum>QFrame::NoFrame</enum>
		 </property>
		</widget>
	   </item>
	  </layout>
	 </widget>
	 <widget class="QWidget" name="tabWidgetPage3">
	  <attribute name="title">
	   <string>Appearance</string>
	  </attribute>
	  <layout class="QGridLayout" name="gridLayout_3">
	   <item row="0" column="0">
		<widget class="QListWidget" name="appearanceListWidget">
		 <property name="frameShape">
		  <enum>QFrame::NoFrame</enum>
		 </property>
		</widget>
	   </item>
	  </layout>
	 </widget>
	 <widget class="QWidget" name="tabWidgetPage4">
	  <attribute name="title">
	   <string>Plugins</string>
	  </attribute>
	  <layout class="QGridLayout" name="gridLayout_4">
	   <item row="0" column="0">
		<widget class="QListWidget" name="pluginListWidget"/>
	   </item>
	  </layout>
	 </widget>
	</widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
